MATURITY.md

```diff
@@ -83,18 +83,18 @@ Mentors and community members are encouraged to contribute to this page and comm
-| CS30 | The project uses documented voting rules to build consensus when discussion is not sufficient.                                                                                                                            | :white_check_mark: The project uses the standard ASF voting rules.                                                                                                                                               |
-| CS40 | In Apache projects, vetoes are only valid for code commits. The person exercising the veto must justify it with a technical explanation, as per the Apache voting rules defined in CS30.                                  | :white_large_square: The project has not used a veto at any point during incubation.                                                                                                                             |
-| CS50 | All "important" discussions happen asynchronously in written form on the project's main communications channel. Offline, face-to-face or private discussions that affect the project are also documented on that channel. | :white_large_square: The project has been making important decisions on the project mailing lists. Minor decisions may occasionally happen during code reviews, which are also asynchronous and in written form. |
+| ID   | Description                                                                                                                                                                                                               | Status                                                                                                                                                                                                         |
+|------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| CS10 | The project maintains a public list of its contributors who have decision power. The project's PMC (Project Management Committee) consists of those contributors.                                                         | :white_check_mark: Committers and PPMC members are listed on the website’s community page, see [newcommitter](https://kyuubi.apache.org/become_committer.html)                                                 |
```

Comment:
as we are here, add one more link [new_ppmc] https://kyuubi.apache.org/become_ppmc_member.html